app/mapping/reads/TrusteeIndividual.scala

```diff
@@ -49,6 +50,13 @@ object TrusteeIndividual extends TrusteeReads[TrusteeIndividual] {
 
   override def trusteeReads: Reads[TrusteeIndividual] = {
 
+    val yesNoDontKnowReads: Reads[YesNoDontKnow] =
```

Comment:
This is a duplication of the reads in app/models/YesNoDontKnow.scala. This also appears to not be used in the reads below as isn't explicitly passed in. Would recommend writing a test around this to convert the old JSON to the model using these reads.